examples/default.js

```diff
@@ -15,7 +15,7 @@ class App extends React.Component {
     });
   }
   render() {
-    return <Pagination onChange={this.onChange} current={this.state.current} total={25} />;
+    return <Pagination onChange={this.onChange} current={this.state.current} total={400} />;
```

Comment:
Add an new example and keep origin example not change : )

Reply:
Updated.